specs/wp-manage-domains-spec.js

```diff
@@ -15,7 +15,6 @@ import ShoppingCartWidgetComponent from '../lib/components/shopping-cart-widget-
 import SidebarComponent from '../lib/components/sidebar-component.js';
 import NavBarComponent from '../lib/components/nav-bar-component.js';
 import MyOwnDomainPage from '../lib/pages/domain-my-own-page';
-import MapADomainComponent from '../lib/components/map-a-domain-component';
```

Comment:
We can remove this whole file I believe as I can't find any other references in the project to it

Reply:
Argh. Yep. I meant to do that. Thanks for reminding me!